Description: The configure's --libdir parameter is used to comply with
 Debian Policy 10.2 which states that not public libraries should be
 installed in subdirectories of /usr/lib. This patch was added in
 order to keep the *-modules and plugins directories in the right
 place.
Forwarded: not-needed
Author: Wences Arana <aranax@debian.org.gt>
Last-Update: 2011-11-05

--- a/libplanner/Makefile.am
+++ b/libplanner/Makefile.am
@@ -4,9 +4,9 @@
 	-I$(top_builddir) \
 	-I$(top_builddir)/libplanner \
 	$(LIBPLANNER_CFLAGS) $(WARN_CFLAGS) \
-	-DSTORAGEMODULEDIR=\""$(libdir)/planner/storage-modules"\" \
-	-DFILEMODULESDIR=\""$(libdir)/planner/file-modules"\" \
-	-DPLUGINDIR=\""$(libdir)/planner/plugins"\"		\
+	-DSTORAGEMODULEDIR=\""$(libdir)/storage-modules"\" \
+	-DFILEMODULESDIR=\""$(libdir)/file-modules"\" \
+	-DPLUGINDIR=\""$(libdir)/plugins"\"		\
 	-DDATADIR=\""$(datadir)/planner"\"
 
 lib_LTLIBRARIES = libplanner-1.la
@@ -64,7 +64,7 @@
 #
 # Storage modules
 #
-storagemoduledir = $(libdir)/planner/storage-modules
+storagemoduledir = $(libdir)/storage-modules
 
 if HAVE_GDA
 sql_library = libstorage-sql.la
@@ -98,7 +98,7 @@
 #
 # File modules
 #
-filemoduledir = $(libdir)/planner/file-modules
+filemoduledir = $(libdir)/file-modules
 
 filemodule_LTLIBRARIES = 			\
 	libmrp-xml.la				\
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -170,7 +170,7 @@
 # Plugins
 #
 
-plugindir = $(libdir)/planner/plugins
+plugindir = $(libdir)/plugins
 
 if HAVE_GDA
 sql_plugin = libsql-plugin.la
